distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -298,7 +325,8 @@ meta value = if is_atom value then Atom.Value value else
         if is_polyglot value then Polyglot.Value value else
             if is_unresolved_symbol value then Unresolved_Symbol.Value value else
                 if is_error value then Error.Value value.catch else
-                    Primitive.Value value
+                    if is_type value then Type.Value value.catch else
```

Comment:
```suggestion
                    if is_type value then Type.Value value else
```

Hm, I'm not really sure why the `.catch` would be here? Is there some reason that I didn't think about?

Reply:
Because I copied previous line ;-) Long live copy paste programming!